Adyen/Core/Payment Methods/Abstract/AnyPaymentMethodDecoder.swift

```diff
@@ -87,6 +89,9 @@ internal enum AnyPaymentMethodDecoder {
             let brand = try? container.decode(String.self, forKey: .brand)
             let isIssuersList = try container.containsValue(.issuers)
 
+            if type == "onlineBanking_CZ" || type == "onlineBanking_SK" {
```

Comment:
OnlineBanking's payment method response contains issuers list and by default we use IssuerListPaymentMethodDecoder() for the payment methods that contains issuers list. So, I had to apply this check because I wanted to use OnlineBankingPaymentMethodDecoder() method.

Reply:
Maybe move it inside of `if isIssuersList { ... }` ?
Then it would be clear that `onlineBanking_CZ` is a special case of "issuers list" ?

Reply:
Very good point. I have moved it inside the  isIssuersList case